index.html

```diff
@@ -9222,6 +9222,240 @@ <h3 id="example-2">
+            </li>
+            <li>Let <var>threshold</var>, <var>knee</var> have the value of the
+            <a>AudioParam</a> of the same name, <a>converted to linear unit</a>
+            sampled at the time of processing of this block (as <a>k-rate</a>
+            parameters).
+            </li>
+            <li>Let <var>ratio</var> have the value of the <code>ratio</code>
+            <a>Audioparam</a>, sampled at the time of processing of this block
+            (as a <a>k-rate</a> parameter).
+            </li>
+            <li>Let <var>attack</var> and <var>release</var> have the value of
+            the <a>AudioParam</a> of the same name, sampled at the time of
+            processing (those are <a>k-rate</a> parameters), converted to
+            sample-frames, based on the sample-rate of the
+            <a>BaseAudioContext</a> this <a>DynamicsCompressorNode</a> is
+            <a href="#associated">associated</a> with.
+            </li>
+            <li>Set <var>attack</var> and <var>release</var> to the
+            multiplication between themselves and the sample-rate for this <a>
+              BaseAudioContext</a>.
+            </li>
+            <li>For each samples <var>input</var> of <var>input signal</var>,
+            execute the following steps:
+              <ol>
+                <li>Let <var>envelope rate</var> be the result of computing the
+                <a href="envelope-rate">envelope rate</a>
+                </li>
+                <li>Let <var>attenuation</var> be the value of putting the
+                absolute value of <var>input</var> <a href="compression-curve">
+                  through the compression curve</a>.
+                </li>
+                <li>Add the difference between <var>attenuation</var> and <var>
+                  detector average</var>, multiplied by the <var>enveloppe
+                  rate</var> to <var>detector average</var>, and clamp
+                  <var>detector average</var> to 1.0.
+                </li>
+                <li>If attacking, add <var>detector average</var> minus
+                <var>compressor gain</var>, multiplied by <var>enveloppe
+                rate</var> to <var>compressor gain</var>, and clamp
+                <var>compressor gain</var> to 1.0.
+                </li>
+                <li>Else, multiply <var>compressor gain</var> by <var>enveloppe
+                rate</var>
+                </li>
+                <li>Compute <var>final gain</var> to be <var>compressor
+                gain</var> multiplied by the return value of <a>computing the
+                makeup gain</a>.
+                </li>
+                <li>Compute <var>metering gain</var> to be <var>final
+                gain</var>, <a href="#linear-to-decibel">converted to
+                decibel</a>.
+                </li>
+              </ol>
+            </li>
+          </ol>
+          <p>
+            The makeup gain is a fixed gain stage that only depends on ratio,
```

Comment:
Oops I forgot to send this one back to the main thread so that authors can read the value back and know how hard the compressor is being hit.